### DISCUSSION TITLE
Is there a good resource for learning about WRF vs WRF-Hydro?

### DISCUSSION THREAD

**Discussion by aputman-usgs** [hytest-org/hytest#531] [Dataset Questions]
I've been reading WRF docs and WRF-Hydro docs, but am trying to figure out if I fully understand the differences / similarities / overlaps. The reading is in an effort to try to understand how the variables output from 'CONUS404' may be similar or different relative to the variables output from 'WRF-Hydro forced with CONUS404-BA', and our specific interest in T and E partitioning in different model / modeling contexts. My current understanding from reading the dataset docs and the NCAR model framework docs is that:

- 'CONUS404' is the result of a WRF run, consisting of fully coupled atmosphere and land surface models downscaling a 30km reanalysis product.
- 'WRF-Hydro forced with CONUS404-BA' is a WRF-Hydro run, consisting of a land surface surface model forced with 'CONUS404-BA'. This modeling framework, in addition to the land surface model, has overland and subsurface routing models, and the land surface model cannot influence the atmosphere. 
- Both CONUS404 and WRF-Hydro forced with CONUS404-BA used Noah-MP as the land surface model.

Roughly, this interpretation might translate to something like:
![image](https://github.com/user-attachments/assets/50ee7fe6-ad8e-44f2-b81e-13280caa046a)

Am I interpreting the model couplings and data relationships roughly correctly? Thanks.

**Comment by rviger** [comment]
Hi Annie, yes, this is all correct. I think it's worth emphasizing that only the components with asterisks next to them in Figure 1.2 are what is used in the "WRF-Hydro forced with CONUS404-BA" run, just in case that escapes the attention of other readers of this thread.

Although both "WRF-Hydro forced with CONUS404-BA" and the CONUS404 runs use Noah/MP, the two model runs do not have the same parameters. Notably, the partitioning of flow within the CONUS404 runs are simplified to meet the downscaling needs, not those of the hydrologic modeling. As a result, equivalent simulated states/fluxes might have differing values in the two data releases. In addition, using CONUS404 for an analysis of the terrestrial components of a water budget is not recommended by the authors

If you have specific questions, like which source to use for transpiration or evaporation, or meaning of differences in these variables from the two sets of results, please let us know!

**Comment by aubreyd** [comment]
@aputman-usgs, you nailed it. For this effort you can think of WRF-Hydro as a standard hydrological model. It can be directly coupled to the WRF atmospheric model to run the full atmo-land-hydro integrated system, but we did not use that approach here given the timing. Both WRF and WRF-Hydro do use the Noah-MP land surface model, but the configurations are a bit different (some different physics options, different spatial resolution, different parameters). The WRF-Hydro model parameters (including some Noah-MP parameters) are calibrated to observed streamflow.

A little additional background on @rviger-usgs 's comment about the C404 water budget: My guess is that ET from CONUS404 would be interesting to look at since that is an important land-atmosphere coupling flux (in other words, if the ET flux was way off, the weather would probably also be off... and the weather looks pretty good!). On the flip side, the WRF-Hydro ET will be (indirectly) impacted by our calibration to streamflow. So we have ET fluxes from roughly the same model (Noah-MP) but constrained by different things (atmospheric prediction vs. hydrologic prediction... ideally we could get all these constraints in the same system!). The Noah-MP E vs. T split I've questioned a bit, but challenging to evaluate. The partitioning between runoff and recharge in CONUS404 is probably not something I would use heavily - it was not designed for hydrologic studies, and it differs quite a bit from other hydrologic models I have looked at.

**Comment by aputman-usgs** [comment]
Hi @rviger-usgs and @aubreyd, 

Thanks for the replies and the info. This is super helpful and clears a lot up for us. 

The deeper motivation for the question and understanding is that our colleagues (via a WRRI grant) at OSU are using NEON site data and stable water isotopes in water vapor to probe/constrain the E vs T partitioning in different land surface and hydrologic models. It's a neat approach and another example of how water isotopes (and other tracers) can really help us with process understanding in modeling contexts that aren't achievable by other means.

It sounds like there may be value in evaluating the T vs E partitioning in both CONUS404 and WRF-Hydro forced with CONUS404-BA, as Noah-MP is parameterized differently in each, and each are tuned / calibrated to a different quantity. I will pass this info along to the team and reply here if we come up with further questions. I'm sure we will.
